src/status_im/ui/screens/wallet/transaction_sent/views.cljs

```diff
@@ -3,38 +3,71 @@
   (:require [status-im.ui.components.react :as react]
             [status-im.ui.components.status-bar.view :as status-bar]
             [status-im.ui.components.icons.vector-icons :as vi]
-            [status-im.ui.screens.wallet.styles :as wallet.styles]
+            [status-im.react-native.resources :as resources]
             [status-im.ui.screens.wallet.transaction-sent.styles :as styles]
-            [status-im.ui.components.styles :as components.styles]
             [re-frame.core :as re-frame]
-            [status-im.ui.screens.wallet.components.views :as components]
             [status-im.i18n :as i18n]
             [status-im.utils.platform :as platform]
             [status-im.ui.components.colors :as colors]))
 
-(defview transaction-sent [& [modal?]]
+(defn- ok-circle []
+  [react/view {:background-color "rgba(0,0,0,0.1)"
+               :width            160
+               :height           160
+               :border-radius    81
+               :align-items      :center
+               :justify-content  :center}
+   [react/view {:background-color colors/white
+                :width            80
+                :height           80
+                :border-radius    41
+                :shadowRadius     4
+                :shadow-offset    {:width 0 :height 2}
+                :shadowOpacity    0.8
+                :shadowColor      "rgba(43, 59, 71, 0.12)"
+                :align-items      :center
+                :justify-content  :center}
+    [vi/icon :icons/tiny-check {:color colors/blue}]]])
+
+(defn- transaction-sent-message []
+  [react/view {:align-items :center}
+   [react/text {:style               styles/transaction-sent
+                :font                (if platform/android? :medium :default)
+                :accessibility-label :transaction-sent-text}
+    (i18n/label :t/transaction-sent)]
+   [react/i18n-text {:style styles/transaction-sent-description
+                     :key   :transaction-description}]])
+
+(defn- bottom-action-button [on-next]
+  [react/touchable-highlight {:on-press on-next
+                              :style {:border-top-width 1
+                                      :border-color     "rgba(255, 255, 255, 0.1)"}
```

Comment:
we have a color in the pallete for this - https://github.com/status-im/status-react/blob/develop/src/status_im/ui/components/colors.cljs#L24

same thing for other colors, if they're in the palette

Reply:
fixed!